lib/guardian/uuid.ex

```diff
@@ -0,0 +1,52 @@
+# Copied from https://github.com/elixir-ecto/ecto/blob/v2.2/lib/ecto/uuid.ex
```

Comment:
@ericmj is this sufficient?

Reply:
Seems good to me, but why we dont use UUID package again?

Reply:
👍

Reply:
For such little and simple code...I definitely prefer copying from Ecto than pulling in a package.